react-keypress.js

```diff
@@ -1,5 +1,5 @@
 var Keypress = require("./bower_components/Keypress/keypress.js");
-var elements = [];
+var elements = new WeakMap;
```

Comment:
I believe you need to do ```new WeakMap()```. Is there any reason to not use ```()```?

Reply:
Just a small style thing I do with initializing new objects without any args. Amended the commit to include them :+1: